servers/rendering/renderer_rd/shaders/tonemap_subpass.glsl

```diff
@@ -0,0 +1,195 @@
+#[vertex]
```

Comment:
Do we need entirely another shader for this? Is not another variant better?

Reply:
That was your suggestion :)

I'm happy to implement it as a variant

Reply:
This is now a shader variant :)